HW5/task3.js

```diff
@@ -0,0 +1,34 @@
+//1
+let word = 'Engineer';
+let vowels = 'aeiou';
+let consonants = 'bcdfghjklmnpqrstvwxyz';
+let countV = 0;
+let countC = 0;
+
+for (let i = 0; i < word.length; i++) {
+    if(vowels.includes(word[i].toLowerCase())) {
+        countV++
+    } else if (consonants.includes(word[i].toLowerCase())) {
+        countC++
+    }
+}
+console.log(`word ${word} contains ${countV} vowels and ${countC} consonants`);
+
+//2
+let str1 = 'you are the best qa ever'
+let code = ''
+let alf = 'abcdefghijklmnopqrstuvwxyz'
+for (let i = 0; i < str1.length; i++) {
+    if (str1[i] === ' ') {    
+        code += ' '
+    }
+    if (str1[i] === 'z') {     
+        code += '*'
```

Comment:
for "z" should be converted to "a" for offset === 1, and "a" to "z" if offset === -1